src/webview/webViewEventHandlers.js

```diff
@@ -180,11 +181,18 @@ const handleLongPress = (
   if (!message) {
     return;
   }
-  const { dispatch, showActionSheetWithOptions, backgroundData, narrow } = props;
+  const {
+    dispatch,
+    showActionSheetWithOptions,
+    backgroundData,
+    narrow,
+    isAnnouncementOnly,
+  } = props;
   showActionSheet(target === 'header', dispatch, showActionSheetWithOptions, _, {
     backgroundData,
     message,
     narrow,
+    isAnnouncementOnly,
   });
```

Comment:
One bonus of using `backgroundData` for the plumbing: it saves us from making a longer and longer list here of things we just pass through without change. (That was part of the origin of the `backgroundData` object in this message-list code.)